Flexible Sync bug fix on multiple anonymous subscriptions. (#7630)
realm/realm-java: CHANGELOG.md

## 10.10.1 (YYYY-MM-DD)

### Enhancements
* None.

### Fixed

> * [RealmApp] Allow defining multiple anonymous subscriptions on a Realm Object.

Reviewer: ```suggestion
* [RealmApp]  Creating multiple anonymous subscriptions for a Realm would throw an exception.
```
